src/const.h

```diff
@@ -113,6 +113,7 @@ enum MagicEffectClasses : uint8_t {
 	CONST_ME_CRITICAL_DAMAGE = 173,
 	// 174 is empty
 	CONST_ME_PLUNGING_FISH = 175,
+	CONST_ME_DRAWINK = 202,
```

Comment:
```suggestion
	CONST_ME_DRAW_INK = 202,
```
?

Reply:
I just wanted to follow DRAWBLOOD pattern 😄